bot/exts/utilities/bookmark.py

```diff
@@ -42,51 +49,38 @@ def build_bookmark_dm(target_message: discord.Message, title: str) -> discord.Em
         return embed
 
     @staticmethod
-    def build_error_embed(user: discord.Member) -> discord.Embed:
-        """Builds an error embed for when a bookmark requester has DMs disabled."""
+    def build_error_embed(message: str) -> discord.Embed:
+        """Builds an error embed for a given message."""
         return discord.Embed(
             title=random.choice(ERROR_REPLIES),
-            description=f"{user.mention}, please enable your DMs to receive the bookmark.",
+            description=message,
             colour=Colours.soft_red
         )
 
     async def action_bookmark(
         self,
         channel: discord.TextChannel,
-        user: discord.Member,
+        member: discord.Member,
         target_message: discord.Message,
         title: str
     ) -> None:
-        """Sends the bookmark DM, or sends an error embed when a user bookmarks a message."""
+        """
+        Sends the given target_message as a bookmark to the member in DMs to the user.
+
+        Send an error embed instead if the member has DMs disabled.
+        """
+        embed = self.build_bookmark_dm(target_message, title)
         try:
-            embed = self.build_bookmark_dm(target_message, title)
-            await user.send(embed=embed)
+            await member.send(embed=embed)
         except discord.Forbidden:
-            error_embed = self.build_error_embed(user)
-            await channel.send(embed=error_embed)
+            error_embed = self.build_error_embed(f"{member.mention}, please enable your DMs to receive the bookmark.")
         else:
-            log.info(f"{user} bookmarked {target_message.jump_url} with title '{title}'")
-
-    @staticmethod
-    async def send_reaction_embed(
-        channel: discord.TextChannel,
-        target_message: discord.Message
-    ) -> discord.Message:
-        """Sends an embed, with a reaction, so users can react to bookmark the message too."""
-        message = await channel.send(
-            embed=discord.Embed(
-                description=(
-                    f"React with {BOOKMARK_EMOJI} to be sent your very own bookmark to "
-                    f"[this message]({target_message.jump_url})."
-                ),
-                colour=Colours.soft_green
-            )
-        )
-
-        await message.add_reaction(BOOKMARK_EMOJI)
-        return message
+            log.info(f"{member} bookmarked {target_message.jump_url} with title '{title}'")
+            return
+        await channel.send(embed=error_embed)
```

Comment:
I think we should move this send into the except block, so it's easier to understand what is happening here.

Reply:
Yea, good idea. [`e5bbd71` (#1049)](https://github.com/python-discord/sir-lancebot/pull/1049/commits/e5bbd7135d147bf054c711b9010546f12c4b22e0)